components/Text/Text.js

```diff
@@ -0,0 +1,48 @@
+import * as styles from './Text.module.scss';
+
+export default function Text({ size, weight, children }) {
+
+  function sizeClass() {
```

Comment:
in these functions I make a switch case on the component props, by that why, the props decide which style would be used